src/neuroconv/datainterfaces/behavior/deeplabcut/deeplabcutdatainterface.py

```diff
@@ -491,15 +491,16 @@ def add_to_nwbfile(
         import warnings
 
         # Use the pose_estimation_metadata_key from the instance if container_name not provided
-        if container_name is None:
```

Comment:
Another one where I got the logic the other way around and you fixed it. Thanks.